locales/en/messages.json

```diff
@@ -3619,12 +3619,12 @@
         "description": "P and D gain tuning slider label"
     },
     "pidTuningDMinRatioSlider": {
-        "message": "D Min Ratio:",
-        "description": "D Min ratio slider label"
+        "message": "D_min gain Drop:",
```

Comment:
Again, this should be consistent with other uses, and 'D Min' seems to be commonly used.